package-lock.json

```diff
@@ -0,0 +1,3692 @@
+{
+  "name": "typescript-sdk",
```

Comment:
Can you remove this file? I think this is generated by mistake and we are not using `npm` in our project.

Reply:
Oops!! Yes, that was my mistake. It is now removed.